atom/browser/ui/file_dialog_mac.mm

```diff
@@ -123,11 +134,14 @@ bool ShowOpenDialog(atom::NativeWindow* parent_window,
                     const base::FilePath& default_path,
                     const Filters& filters,
                     int properties,
+                    const std::string& message,
                     std::vector<base::FilePath>* paths) {
   DCHECK(paths);
   NSOpenPanel* dialog = [NSOpenPanel openPanel];
 
-  SetupDialog(dialog, title, button_label, default_path, filters);
+  SetupDialog(dialog, title, button_label, default_path, filters, message,
+          // NSOpenPanel does not support name_field_label and shows_tag_field
+          "", false);
```

Comment:
Looks like the alignment here is off, can you align this with the `dialog` argument on line 142?